### PR TITLE
BZ #956256 and #956261: added a link to candlepin-cert-consumer package .that omits fqdn of the server and the version from the file name.

### PR DESCRIPTION
**Pull request by dmitri-d**
Redo of the PR from Katello repository...

### CODE REVIEW

**Comment by dmitri-d** [comment]
@xsuchy, @thomasmckay: ping

**Comment by thomasmckay** [comment]
ACK

File: modules/certs/manifests/config.pp [hunk 1]
```diff
@@ -118,11 +118,19 @@
     require   => [Exec["generate-candlepin-certificate"], File["${ssl_build_path}/rhsm-katello-reconfigure"], File["${katello::params::configure_log_base}"]]
   }
 
+  exec { "create-candlepin-consumer-certificate-link":
```
Comment (iNecas) [inline modules/certs/manifests/config.pp]: I would consider using file { "link_path": ensure => link} instead: its more idiomatic for puppet

Comment (iNecas) [inline modules/certs/manifests/config.pp]: http://docs.puppetlabs.com/guides/style_guide.html#symbolic-links

**Comment by dmitri-d** [comment]
@iNecas: considered, thought it was a good idea, changed.

**Comment by iNecas** [comment]
ACK

**Pull request merged**